packages/rest/package.json

```diff
@@ -44,6 +44,7 @@
     "lodash": "^4.17.15",
     "on-finished": "^2.3.0",
     "path-to-regexp": "^6.1.0",
+    "p-event": "^4.1.0",
```

Comment:
Do we need to add `@types/p-event` to our dev dependencies?

Reply:
No, `p-event@4.1.0` has self-contained `d.ts`.